reconcile/terraform_cloudflare_dns.py

```diff
@@ -0,0 +1,292 @@
+import logging
+import sys
+from typing import (
+    Any,
+    Callable,
+    Dict,
+    List,
+    Mapping,
```

Comment:
These should be imported `from collection.abc import (...)` for [PEP-585](https://peps.python.org/pep-0585/) compliance.

Reply:
Further, I think `List` and `Dict` can be omitted, as `list` and `dict` are also valid.